pages/about.html

```diff
@@ -43,7 +44,7 @@
     <section class="about-us-wrapper">
       <div class="about-us-content column">
         <div class="about-us__image">
-          <img src="/assets/images/website-images/logo-with-background.png" />
+          <img src="/assets/images/website-images/logo-with-background.png"  alt="logo"/>
```

Comment:
same as previous comment, we'll need to improve from just 'logo' to something more descriptive